pkg/kubefed2/disable.go

```diff
@@ -104,17 +110,35 @@ func NewCmdTypeDisable(cmdOut io.Writer, config util.FedConfig) *cobra.Command {
 
 	flags := cmd.Flags()
 	opts.GlobalSubcommandBind(flags)
+	opts.CommonSubcommandBind(flags, federationGroupUsage, targetVersionUsage)
 	opts.Bind(flags)
 
 	return cmd
 }
 
 // Complete ensures that options are valid and marshals them if necessary.
 func (j *disableType) Complete(args []string) error {
-	if len(args) == 0 {
-		return errors.New("NAME is required")
+	j.enableTypeDirective = enable.NewEnableTypeDirective()
+	fd := j.enableTypeDirective
```

Comment:
`fd` is probably confusing, `etd` or just `td` would be better for a reader with no prior code knowledge.. :-)